### DISCUSSION TITLE
Status of rewrite?

### DISCUSSION THREAD

**Discussion by mikenerone** [Bubz43/ep2e#12] [General]
May I ask how things are progressing for the rewrite mentioned in the README? I'm new to both EP and Foundry, and more information would help in deciding which of the two game systems to invest time into (unfortunately, the [one post about the topic](https://www.foundryvtt-hub.com/news/system/lets-explore-eclipse-phase-on-foundry-vtt/) is 3.5 years old and likely to be out of date).

**Comment by Bubz43** [comment]
I was really hoping for a playable version of the rewrite by the end of 2024, but unfortunately the last 3 months didn't have much progress for one reason or another. As such a playable version of the rewrite is at minimum 2-3 months away.  I do still do bug fixes and update the version on this repo to work with the latest major foundry versions, and while I haven't seen what I'll need to do for the upcoming foundry version its likely I'll be able to update it for that as well. The rewrite is going to be different enough from this version though that it will be basically learning a new system from scratch as well.

**Reply by mikenerone** [reply]
Thanks so much for the quick response! Obviously real life doesn't always facilitate our wishes. I'm glad to hear that work is still progressing on it.

I need to pick which of the EP2 game-system plugins for Foundry I'm going to use for the time being (I'm really looking forward to your next-generation one, but as you say it's a complete departure from the current version, the "invested learning curve" argument doesn't so much apply). I'm going to be starting an EP2 campaign soon, and undertaking so many new TTRPG aspects at once, I'm a bit overwhelmed (new to EP2, new to Foundry, haven't GM'd a game in, like, decades), and I don't have the time I wish I did to properly deep-dive each of them to make an educated decision on my own, so any advice you can offer would be appreciated.

BTW, you mentioned that maintenance has included Foundry compatibility updates, so it sounds like it should work with v12. The README only says that Foundry v11 is supported, so I hadn't actually tried, but I will now. If so, you might want to update the README. EDIT: Oh, now I see in the manifest that v12 actually _is_ supported - I should have thought to look there.

**Reply by Bubz43** [reply]
Good call on the README, forgot about having the foundry version note in there. 

As for advice I'm quite possible not the best person to ask, as I've been in a perpetual state of building the system, rather than actually _using_ the system. I don't know much about the other system other than it deviates for EP in some ways, and add its own flavor, but I don't actually know in what ways/how. As for mine, it mostly starts falling apart when you reach the trickier EP stuff like swarms, flexbots and in-depth mesh actions. Also I didn't really put in any explanations for anything, so it either magically works...or it doesn't. 

You could try joining the discord also linked in the README as there are people there who are actively using the system and can give you a better answer to questions you might have than I can.
